### PR TITLE
Added discord_controller file for testing discord integration

### PR DESCRIPTION
**Pull request by Samueljbk**
Hard coded for my ip because ran into trouble adapting it to the discover_bulbs() function, also not going to work because of the bot token being invalidated once made public but works on my machine anyway lol

### CODE REVIEW

File: src/be/discord_controller.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import discord
+from discord.ext import commands
+from yeelight import Bulb
+
+TOKEN = "MTA5MzQyNjQ4MDYxMDAyMTQ0OA.G1JUcC.Gbz2s1AuNll4Qj4PDFCJVcUh1u33iEzDRhPKow"
+bulb_ip = "192.168.86.81"
+
+intents = discord.Intents.default()
+intents = discord.Intents.default()
```
Comment (jerome3o) [inline src/be/discord_controller.py:11]: dupe line

Comment (jerome3o) [reply src/be/discord_controller.py:11]: looks like you've accidentally added this line twice, can probably be removed

File: src/be/discord_controller.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import discord
+from discord.ext import commands
+from yeelight import Bulb
+
+TOKEN = "MTA5MzQyNjQ4MDYxMDAyMTQ0OA.G1JUcC.Gbz2s1AuNll4Qj4PDFCJVcUh1u33iEzDRhPKow"
```
Comment (jerome3o) [inline src/be/discord_controller.py]: add `import os` to your imports at the top

replace with:
```
TOKEN = os.environ.get("DISCORD_TOKEN")
bulb_ip = os.environ.get("BULB_IP")
```

And we can go over environment variables at some point - would be good for you to have a read up and we can discuss/fill in the details.

File: src/be/discord_controller.py (hunk 1 above)
Comment (Samueljbk) [reply src/be/discord_controller.py:11]: Done

File: src/be/discord_controller.py (hunk 2 above)
Comment (Samueljbk) [reply src/be/discord_controller.py]: Sounds good - Done

**Review by jerome3o — APPROVED** [review]

**Pull request merged**